packages/FAQ/docs/introducing-smart-invoice.md

```diff
@@ -0,0 +1,28 @@
+---
+title: What is Smart Invoice?
+---
+
+
+Smart Invoice is a feature rich invoicing tool that facilitates payments B2B on-chain; Smart Invoice establishes a contractual relationship whereby two parties who are engaged in business with one another can create a set of shared enforceable expectations. v1 features are as follows: 
```

Comment:
I totally agree with that sentiment on the front of "contracts", it could possibly deter people from using our product so i'll remove that.  B2B is probably not quite correct, i chose the verbage there because it was easiest to explain.  HAHA on the business-y language, i'll resculpt with less boring text